Index: b/debian/rules
===================================================================
--- a/debian/rules
+++ b/debian/rules
@@ -1057,12 +1057,6 @@
 	: # Tkinter library files shipped in python3-tk
 	rm -rf $(d)/usr/lib/python$(VER)/tkinter
 
-	: # distutils shipped in python3-distutils
-	rm -rf $(d)/usr/lib/python$(VER)/distutils/*
-	: # ... except for distutils.version
-	cp Lib/distutils/{__init__,version}.py \
-		$(d)/usr/lib/python$(VER)/distutils/.
-
 	: # lib2to3 shipped in python3-lib2to3
 	rm -rf \
 		$(d)/usr/bin/2to3-$(VER) \
